src/hfold/Parser.hs

```diff
@@ -0,0 +1,55 @@
+module Hfold.Parser where
+
+import Data.Monoid ((<>))
+import Data.Text (Text)
+import qualified Data.Text as T
+import Text.ParserCombinators.Parsec
+
+import Hfold.Types
+
+parseFile :: Text -> [Content]
+parseFile t = case parse parseContent "content" (T.unpack t) of
```

Comment:
This could also be `either (const []) id $ parse parseContent "content" (T.unpack t)`.